sql-analyzer/src/com/cubrid/analyzer/SQLAnalyzer.java

```diff
@@ -0,0 +1,320 @@
+package com.cubrid.analyzer;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URI;
+
+import com.cubrid.database.DatabaseManager;
+import com.cubrid.parser.SqlMapParser;
+import com.cubrid.validator.SQLValidator;
+
+public class SQLAnalyzer {
+	private final String SUMMARY_FILE_NAME = "summary.log";
+	private final String SQLMAP_PATH = "sqlmap";
+
+	private String resultFilePath = null;
+	private File resultFile = null;
+
+	private URI rootPath = null;
+	private BufferedWriter writerSummary = null;
+
+	private int count = 0;
+	private int totalCount = 0;
+	private int totalSuccessCount = 0;
+	private int totalErrorCount = 0;
+	private int subTotalCount = 0;
+	private int subTotalSuccessCount = 0;
+	private int subTotalErrorCount = 0;
+
+	private boolean isBeforeFile = false;
+
+	private DatabaseManager databaseManager = null;
+
+	public static void main(String[] args) {
+		SQLAnalyzer sqlAnalyzer = new SQLAnalyzer();
+		sqlAnalyzer.start();
+	}
+
+	public SQLAnalyzer() {
+		databaseManager = new DatabaseManager();
+
+		/* debug */
+		databaseManager.initPseudoConnect();
```

Comment:
I think that initPesudoConnect() dosen't connect to DBMS and only check syntax validation using by jni.
please support the way checking syntax and semantic error as well as only syntax error.